end-to-end-tests/cypress/integration/organisations.js

```diff
@@ -5,12 +5,24 @@ const baseUrl = Cypress.config().baseUrl;
 describe("login", function () {
   // NOTE: user only has one organisation associated so,
   //       it can not view a list of organisations
-  it.skip("viewing B1T organisation details ", function () {
-    cy.signIn()
-      .visit(baseUrl);
+  it("viewing B1T organisation details ", function () {
+    const params = {
+      auth: {
```

Comment:
might be worth putting this into a re-usable helper function but maybe in another pR

Reply:
The longer term plan is to remove cypress for something else. Maybe the ground work from DTTP gets moved into this project. We don't really add to these at the moment so I'm not too concerned around quality of this as hopefully it will be binned soon.